lib/src/slices/data_bkg/data_bkg_service.dart

```diff
@@ -13,265 +12,156 @@ import '../api_app_data/model/api_app_data_model.dart';
-      _log.fine('Gmail fetch aborted. No Google Account.');
-      return;
-    }
-    ApiAppDataModel? appDataGmailLastFetch =
-        await _apiAppDataService.getByKey(ApiAppDataKey.gmailLastFetch);
-    int? gmailLastFetchEpoch = fetchAll || appDataGmailLastFetch == null
-        ? null
-        : int.parse(appDataGmailLastFetch.value);
-    if (force == true ||
-        gmailLastFetchEpoch == null ||
-        DateTime.now().subtract(Duration(days: 1)).isAfter(
-            DateTime.fromMillisecondsSinceEpoch(gmailLastFetchEpoch))) {
-      DateTime run = DateTime.now();
-      String activeCategory =
-          (await _apiAppDataService.getByKey(ApiAppDataKey.gmailCategory))
-                  ?.value ??
-              model.gmailCategoryList[0];
-      int categoryStartIndex = model.gmailCategoryList.indexOf(activeCategory);
-      int totalCats = model.gmailCategoryList.length;
-      for (int i = categoryStartIndex; i < totalCats; i++) {
-        String category = model.gmailCategoryList[i];
-        String query = _gmailBuildQuery(gmailLastFetchEpoch, category);
-        await _checkGmailFetchList(query: query);
-        int nextCatIndex = model.gmailCategoryList.indexOf(category) + 1;
-        String nextCat = nextCatIndex == totalCats
-            ? model.gmailCategoryList[0]
-            : model.gmailCategoryList[nextCatIndex];
-        await _apiAppDataService.save(ApiAppDataKey.gmailCategory, nextCat);
-      }
-      await _apiAppDataService.save(
-          ApiAppDataKey.gmailLastFetch, run.millisecondsSinceEpoch.toString());
-      await _apiAppDataService.save(ApiAppDataKey.gmailPage, '');
-      _log.fine(
-          'Gmail fetch completed on: ' + DateTime.now().toIso8601String());
-    }
-  }
-
-  Future<void> _checkGmailFetchList({String query = ''}) async {
-    String? page;
-    ApiAppDataModel? appDataGmailPage =
-        await _apiAppDataService.getByKey(ApiAppDataKey.gmailPage);
-    page = appDataGmailPage?.value;
-    return _checkGmailFetchListPage(query: query, page: page);
+        this._apiEmailMsgService = apiEmailMsgService {
+    getExistingAccounts().then((_) => fetchAllData());
   }
 
-  Future<void> _checkGmailFetchListPage({String? page, String? query}) async {
-    DataBkgModelPage<String> res = await _apiGoogleService.gmailFetch(
-        query: query, page: page, maxResults: 5);
-    if (res.data != null) {
-      List<String> known =
-          (await _apiEmailMsgService.getByExtMessageIds(res.data!))
-              .map((message) => message.extMessageId!)
-              .toList();
-      List<String> unknown =
-          res.data!.where((message) => !known.contains(message)).toList();
-      await _checkGmailFetchMessage(unknown);
-    }
-    await _apiAppDataService.save(ApiAppDataKey.gmailPage, res.next ?? '');
-    if (res.next != null)
-      return _checkGmailFetchListPage(page: res.next, query: query);
+  Future<void> getExistingAccounts() async {
```

Comment:
this should be called loadAccounts, it does not "get" anything since it returns void